addon/components/nypr-accounts/membership-card.js

```diff
@@ -8,9 +8,9 @@ export default Component.extend({
   layout,
   session: service(),
   pledgeManagerEnabled: true,
-  pledgeRedirectMessage: "Manage your account and find your tax documents in a new place.",
-  pledgeRedirectUrl: "https://pledge.wnyc.org/user",
-  pledgeRedirectUrlText: "Manage My Account",
+  customStatusMessage: null,
```

Comment:
small thing, but using `null` where not required is a bit of an anti-pattern in some circles. The reason for this is that `null` has no properties, and so executing something like `null.substring(0, 1)` will throw an error, whereas `"".substring(0, 1)` will not, but `""` is also `falsy` so it should work similarly with your `#if` statements.

Reply:
Makes sense